### PR TITLE
release: `v11.2.0`

### PR DESCRIPTION
**Pull request by georglauterbach**
# Description

I know, this is a release PR, but:

I introduced the "Keep a Changelog"-format which helps me in **not** spending this much time on changelogs in the future. There is a another PR (#2789) that will update the documentation before v11.2.0 is released, which focuses on adding the entry to the changelog before merging a relevant PR.

Releasing is always a **huge** efforts when I am writing the changelog because of the many manual adjustments. This is unacceptable and needs to change. The format change is the solution I think.

I already prepared a draft for v11.2.0, have a look :D

Currently waiting on:

- [x] #2789
- [x] #2786
- [x] #2796
- [x] #2797

Feature freeze is now in place until **12 Oct 2022**.

## Type of change

- [x] Release

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (README.md or the documentation under `docs/`)
- [x] If necessary I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by wernerfred — APPROVED** [review]
I like it! Writing the release information was always a manual nightmare.
Just curious: what speaks against taking the release notes that github can autogenerate for the changelog? It is not that structured but i like the fact that every change is included (even if very small) and every change is linked to its PR.

**Comment by georglauterbach** [comment]
> I like it! Writing the release information was always a manual nightmare.

Indeed 🙏🏼

> Just curious: what speaks against taking the release notes that github can autogenerate for the changelog? It is not that structured but i like the fact that every change is included (even if very small) and every change is linked to its PR.

Nothing really, but I like the way that "Keep a Changelog" structures the information. I decided against just copying, because

1. I would put the (automatically generated) list of all PRs into the [Releases](https://github.com/docker-mailserver/docker-mailserver/releases) page so no information is lost
2. Have the `CHANGELOG.md` show information in a nice, structured way

**Comment by georglauterbach** [comment]
NICE UPDATE: Take a look at the latest manual run I started: <https://github.com/docker-mailserver/docker-mailserver/actions/runs/3100146489>

Image build time is 1:15 min :D Nice! Caches and generic workflows seem to work! 🙌🏼

**Comment by georglauterbach** [comment]
@casperklein feel free to bring up your PR about the F2B/Dovecot packages - no time pressure with this release.

**Comment by georglauterbach** [comment]
@casperklein do you want to merge #2808 before we do the freeze & release this?

**Comment by georglauterbach** [comment]
Alright, feature-freeze is in-place.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,66 @@
...
+
+### Summary
+
+This release features a lot of small and medium-sized changes, many related to how the image is build and tested during CI. The build now requires Docker Buildkit as the ClamAV Signatures are added via `COPY --link ...` during build-time. Moreover, the build is now multi-stage. `ENABLE_LDAP` is now deprecated.
+
+### Added
+
+- **documentation**: improve cron tasks documentation and fix link in documentation
+- **documentation**: added link to brakkee.org for setup of docker-mailserver on Kubernetes
+- **CI**: better build caching for CI
```
Comment (casperklein) [inline CHANGELOG.md:19]:
```suggestion
- **CI**: better build caching for CI ([#2742](https://github.com/docker-mailserver/docker-mailserver/pull/2742))
```

There are no more links to the corresponding PR, was that on purpose? When I read any change logs, sometimes I like to check, what exactly have changed. Having a direct link to the PR makes it easy.

Comment (polarathene) [reply CHANGELOG.md:19]: > When I read any change logs, sometimes I like to check, what exactly have changed. Having a direct link to the PR makes it easy.

:100:

**Review by casperklein — APPROVED** [review]
No issues in my setup 👍

**Pull request merged**